tests/embind/test_finalization.cpp

```diff
@@ -0,0 +1,27 @@
+#include <emscripten/bind.h>
+#include <iostream>
+#include <memory>
+
+class Foo {
+  std::string mName;
+
+public:
+  Foo(std::string name = "Constructed from JS") : mName(name) {}
```

Comment:
I'd rather pass that string explicitly from JS side for clarity in this case - it can be hard to read the code and understand why it's the default value for the main C++ constructor.